lib/LMS.class.php

```diff
@@ -2428,6 +2428,20 @@ public function GetUniqueInstallationID()
         return $uiid;
     }
 
+    public function ReplaceInstanceLabelSymbols($text)
+    {
+        $search = array(
+            '%hostname',
+            '%dbversion',
+        );
+        $replace = array(
+            gethostname(),
+            $this->DB->GetVersion(),
+        );
+
+        return str_replace($search, $replace, $text);
```

Comment:
Warto tu tworzyć zmienne? Nie można od razu inline wywołać w str_replace()?

Reply:
Zastanawiałem się nad tym akurat. Wydaje mi się, że kod prostszy i szybszy. Raz uruchamiasz str_replace. 
No ale przez to nie działają zagnieżdzenia jeśli np. %hostname zawierałaby którąś z pozostałych zmiennych...
Jak ma zostać?

Reply:
Zgodnie z prośbą db9044570c5d17a146c29807c68fb4566df2edf4